src/App.js

```diff
@@ -14,6 +14,7 @@ import { Helmet } from 'react-helmet';
 import { availableLanguageOrNull } from './i18n/utils';
 import Shopify from './components/Integrations/Shopify/Shopify';
 import SignupConfirmation from './components/Signup/SignupConfirmation';
+import SubscriberHistory from './components/SubscriberHistory/SubscriberHistory';
```

Comment:
If we were saying that `SubscriberHistory` is a report, wouldn't it be better to add into reports folder?

Reply:
Yes i think the same. I talked yesterday about it. But maybe need to refactor the reports too because we have a "reports" folder with `Reports.js` file that we use in Advanced reports. But maybe can change in this pr too